DataFormats/BTauReco/interface/TemplatedSecondaryVertexTagInfo.h

```diff
@@ -58,7 +58,7 @@ class TemplatedSecondaryVertexTagInfo : public BaseTagInfo {
 
         struct VertexData {
                 VTX                             vertex;
-                Measurement1D                   dist2d, dist3d;
+                Measurement1D                   dist1d,dist2d, dist3d;
                 GlobalVector                    direction;
 		
 		// Used by ROOT storage
```

Comment:
Hi Sarmad,
Please update the version in this file (from 11 to 12 in two places).
I will need some instructions from Chris on how to update classes_def.xml with checksums included.
If I understand correctly, the change in classes_def.xml is not required at this point.